src/site-type/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { memo } from 'react';
+import { __ } from '@wordpress/i18n';
+import { ToggleDropdown } from '../index.js';
+
+
+const SiteType = ({ value, onClick }) => {
+	return (
+		<ToggleDropdown
+			className="stc-site-type-filter"
+			value={value}
+			options={[
+				{
+					id: '',
+					title: __('All', 'astra-sites'),
```

Comment:
I think we can create a separate task for the text-domain in which we'll replace all the text domains.

Reply:
For now, the text domains could be removed. Just `__('All')` is also okay. Although I agree this can be done in a separate task.